ddtrace/opentracer/span.py

```diff
@@ -49,14 +22,12 @@ def __init__(self, tracer, context, operation_name):
 
         super(Span, self).__init__(tracer, context)
 
+        self.finished = False
```

Comment:
I guess all attributes must be internal, I would say to keep all attributes internal because this is only the implementation of an API